mkfs/mkfs.c

```diff
@@ -12,6 +12,9 @@
 #include <limits.h>
 
 #include <region.h>
+#include <storage.h>
+
+#define BOOTFS_SIZE (8 * MB)
```

Comment:
Minor: parameters like these are typically stored in src/runtime/uniboot.h.

Reply:
OK, I moved it there.